packages/messenger-widget/src/components/ConfigureProfile/bl.tsx

```diff
@@ -65,22 +66,24 @@ export const getEnsName = async (
     account: Account,
     setEnsNameFromResolver: Function,
 ) => {
-    if (ethAddress) {
-        const isAddrEnsName = account.ensName?.endsWith(
-            globalConfig.ADDR_ENS_SUBDOMAIN(),
-        );
-        const name = await mainnetProvider.lookupAddress(ethAddress);
-        if (name && !isAddrEnsName) {
-            const hasProfile = await hasUserProfile(mainnetProvider, name);
-            const dm3ProfileRecordExists = await checkEnsDM3Text(
-                mainnetProvider,
-                name,
+    try {
+        if (ethAddress) {
+            const isAddrEnsName = account.ensName?.endsWith(
+                globalConfig.ADDR_ENS_SUBDOMAIN(),
             );
-            hasProfile &&
-                dm3ProfileRecordExists &&
-                setEnsNameFromResolver(name);
+            const name = await mainnetProvider.lookupAddress(ethAddress);
+            if (name && !isAddrEnsName) {
+                const hasProfile = await hasUserProfile(mainnetProvider, name);
+                const dm3ProfileRecordExists = await checkEnsDM3Text(
+                    mainnetProvider,
+                    name,
+                );
+                hasProfile &&
+                    dm3ProfileRecordExists &&
+                    setEnsNameFromResolver(name);
+            }
         }
-    }
+    } catch (error) {}
```

Comment:
Please add at least a log to make the error visible

Reply:
Ok will add logs